### PR TITLE
Reactor UI Control Rod Guage Tweak

### PR DESCRIPTION
**Pull request by amylizzle**
## About the PR 
I changed the control rod insertion gauge to read better for left-to-right readers. 

## Why's this needed? 
Looks better.

### CODE REVIEW

**Comment by amylizzle** [comment]
!merge_upstream

**Comment by pali6** [comment]
!merge_upstream

**Comment by pali6** [comment]
pog

**Review by Tarmunora — APPROVED** [review]
DM changes look fine

File: code/obj/nuclearreactor/nuclearreactor.dm [hunk 1]
```diff
@@ -280,14 +280,14 @@
...
-		rads -= 10
+		src.AddComponent(/datum/component/radioactive, min(rads*3, 100), TRUE, FALSE, 5)
+		rads -= 5
 
 		if(rads <= 0)
 			return
 
-		for(var/i = min(round(rads/2),50),i>0,i--)
-			shoot_projectile_XY(src, new /datum/projectile/neutron(min(rads*2,100)), rand(-10,10), rand(-10,10)) //for once, rand(range) returning int is useful
+		for(var/i = min(ceil(rads/2),50),i>0,i--)
```
Comment (pali6) [inline code/obj/nuclearreactor/nuclearreactor.dm:289]: Formatting but mostly I just want a random commit so checks rerun.
```suggestion
		for(var/i = min(ceil(rads / 2), 50), i>0, i--)
```

**Comment by pali6** [comment]
not pog (!merge_upstream doesn't work)

File: tgui/packages/tgui/components/RoundGauge.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ export const RoundGauge = props => {
     maxValue = 1,
     ranges,
     alertAfter,
+    alertBefore,
```
Comment (mordent-goonstation) [inline tgui/packages/tgui/components/RoundGauge.js:26]: Is this included in the upstream tgui component? If not, possibly PR in there (and link here for a use-case)? Prefer not deviating from upstream tgui for core components if possible.

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]
tgui otherwise looks fine.

File: tgui/packages/tgui/components/RoundGauge.js (hunk 2 above)
Comment (amylizzle) [reply tgui/packages/tgui/components/RoundGauge.js:26]: You know I didn't even think about the upstream. I'll PR it.

Comment (amylizzle) [reply tgui/packages/tgui/components/RoundGauge.js:26]: So fun fact, I went to go do this and it's already done in the upstream in almost exactly the way I did it.

**Comment by amylizzle** [comment]
Okay, given the TGUI mess, I'm going to atomise the balance changes from the UI tweaking. This will remain the UI tweaking PR.

**Comment by ZeWaka** [comment]
todo soon

**Review by ZeWaka — APPROVED** [review]
part of PR i already cleared

**Pull request merged**